### PR TITLE
EBF-209: Add `<TopNavigation>` component

### PR DESCRIPTION
**Pull request by obyknovenius**

### CODE REVIEW

File: app/styles/ebf-design-system/themes/telekom/components/_navbar.scss [hunk 1]
```diff
@@ -430,6 +438,51 @@
...
+
+.navbar-header-secondary-light {
+  @extend .navbar-light;
+  @extend .navbar-secondary;
+}
+
+.navbar-header-secondary-dark {
+  @extend .navbar-dark;
+  @extend .navbar-secondary;
+}
```
Comment (obyknovenius) [inline app/styles/ebf-design-system/themes/telekom/components/_navbar.scss]: So many `@extend`s. Is there a smarter way?

Comment (vladaspasic) [reply app/styles/ebf-design-system/themes/telekom/components/_navbar.scss]: Maybe a `mixin` is a better approach here

**Comment by obyknovenius** [comment]
@vladaspasic I'm going to merge this PR as-is for now. We can improve the implementation later on.

File: addon/components/top-navigation/index.hbs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<BsNavbar
+  @type={{@type}}
+  @backgroundColor="nav-type"
```
[the remark this answers is not in the document]
Comment (vladaspasic) [reply addon/components/top-navigation/index.hbs:3]: I am not sure what does this property do. It is a bit difficult to deduct as I would expect that when i define a value for a `backgroundColor` to be a color name, not a `nav-type`. Just a bit confusing

**Pull request merged**